avg.py

```
@@ -0,0 +1,15 @@
+
+
+def calculate_average(num1, num2, num3):
+    sum_of_numbers = num1 + num2 + num3
+    average = sum_of_numbers / 3
```

Ruff (0.12.2)

Local variable `average` is assigned to but never used

Remove assignment to unused variable `average`

(F841)